controllers/index.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-console */
+const movies = require('../movies')
+
+// controller for all moves //
+const getAllMovies = (request, response) => {
+  return response.send(movies)
+}
+
+// controler for getByTitle//
+const getByTitle = (request, response) => {
+  const { title } = request.params
+  const specificTitle = movies.filter(movie => {
+    console.log(movie.title)
+
+    return movie.title.toLowerCase().includes(title.toLowerCase())
+  })
+
+  return response.send(specificTitle)
+}
+
+//controller for getByDirector//
+const getByDirector = (request, response) => {
+  const directorFound = movies.filter(movie => { return movie.directors.some(director => {
+    return director.toLowerCase().includes(request.params.directors.toLowerCase())
+  })
+  })
```

Comment:
minor formatting

Reply:
and to satisfy the API as per documentation, you can technically combine the getByDirectory and getByTitle into one function.

/movies/<titleOrDirector>